python/ql/src/semmle/python/frameworks/Django.qll

```diff
@@ -16,7 +16,284 @@ private import semmle.python.regex
  * Provides models for the `django` PyPI package.
  * See https://www.djangoproject.com/.
  */
-private module Django {
+module Django {
```

Comment:
Since you say in one of the commit messages: 

>I'm not quite sure whether the modules `Django::Views` and `Django::Forms` are actually helpful, or whether we should just have their modules available as
`Django::View`, `Django::Form`, and `Django::Field`...

then perhaps it would be better to keep this `private` until the point where it actually becomes necessary to open up these classes?
```suggestion
private module Django {
```

Reply:
I was mainly mirroring the setup we did for flask (which also has this wrapper module), but for now, we can do this :+1: (allowing us to fix up things later down the road)